Classes/Private/Manager/Objects/OCTActiveFile+Variants.h

```diff
@@ -0,0 +1,58 @@
+//
+//  OCTActiveFile+Variants.h
+//  objcTox
+//
+//  Created by stal on 9/7/2015.
+//  Copyright © 2015 Zodiac Labs. All rights reserved.
+//
+
+#import "OCTSubmanagerFiles.h"
+#import "OCTActiveFile.h"
+
+#define AVERAGE_SECONDS 10
```

Comment:
Please use constants instead of defined
```objc
static const NSUInteger kOCTActiveFileAverageSeconds = 10;
```
Also this constant can be moved to `.m` file, in this case naming can be simpler
```objc
static const NSUInteger kAverageSeconds = 10;
```